python-package/xgboost/core.py

```diff
@@ -558,8 +558,17 @@ def __init__(
             Whether print messages during construction
         feature_names : list, optional
             Set names for features.
-        feature_types : list, optional
-            Set types for features.
+        feature_types :
+
+            Set types for features.  When `enable_categorical` is set to `True`, string
+            "c" represents categorical data type.  For numerical data, it can be one for
+            the following:
+
```

Comment:
I don't understand what happens when I change the numerical types, is there more documentation somewhere?

Reply:
Not at the moment, it's only used for text model dump.  Actually, I have been think if it's possible to remove them along with the `fmap` parameter. (they are the same thing).

Reply:
I removed the doc for numerical data.  Let's just keep it as it's for now.